core-ui/src/components/Extensibility/components/Columns.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 
 import { Widget } from './Widget';
 
+import './InlineDisplay.scss';
+import { isNil } from 'lodash';
+
 export function Columns({ structure, ...props }) {
+  const inline = isNil(structure.inline)
+    ? props.inlineContext
```

Comment:
and then go with 
```suggestion
    ? inlineContext
```
here?

Reply:
(and add `inlineContext={inlineContext}` to the returned Widget?)